app/UiTiDv1/Listeners/CreateConsumers.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UiTiDv1\Listeners;
+
+use App\Domain\Integrations\Events\IntegrationCreated;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\UiTiDv1\Repositories\UiTiDv1ConsumerRepository;
+use App\UiTiDv1\UiTiDv1ClusterSDK;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+final class CreateConsumers implements ShouldQueue
```

Comment:
Maybe we should add some logging like is done for the Isnightly integration and also log failed jobs?
If so this can also be done for the Auth0 integration.

Reply:
I'll take a look at how it's done for Insightly now. But if we also need to add it for Auth0, I think it's better to do it in a follow-up PR? (I exactly followed the approach of the Auth0 integration in this one)

Also, does Horizon not log the failed jobs?

Reply:
Adding this in another pull request is perfect.

Horizon does indeed log it, but I thought it could be useful if we would at extra functionality on top of logging.
If for example, dev-ops set monitors on logging files everything is in a central place.

Reply:
Alright, I've created a follow-up ticket for it: https://jira.uitdatabank.be/browse/PPF-106